js_modules/dagit/packages/core/src/instance/BackfillRow.tsx

```diff
@@ -131,10 +131,14 @@ export const BackfillRow = ({
         )}
       </td>
       <td>
-        {counts ? (
-          <BackfillRunStatus backfill={backfill} counts={counts} statuses={statuses} />
+        {backfill.isValidSerialization ? (
+          counts ? (
+            <BackfillRunStatus backfill={backfill} counts={counts} statuses={statuses} />
+          ) : (
+            <LoadingOrNone queryResult={queryResult} />
+          )
         ) : (
-          <LoadingOrNone queryResult={queryResult} />
+          <p>A partitions definition has changed since this backfill ran.</p>
```

Comment:
Not sure if this should say "partition definition" or "partitions definition"?

Reply:
Hmm... our definitions objects are called `PartitionsDefinition` so I lean toward following this